### DISCUSSION TITLE
Can we use GGUF model files with this extension?

### DISCUSSION THREAD

**Discussion by tusharbhutt** [imrayya/stable-diffusion-webui-Prompt_Generator#26] [Q&A]
I haven't downloaded any, but wanted to make sure the extension can support the format

**Comment by imrayya** [comment]
It's been a while since I touched the code base but it just uses GPT2 and its close "relatives" (such as distilGPT2).

As for implementing it, I am on holiday until the end of the month so I won't be able to test/code it until then. Feel free to wait (I'm not sure if it would work) or put in a pull request. 

TLDR: It won't for now
